todo-mvc/src/containers/TodoAppContainer.ts

```diff
@@ -0,0 +1,44 @@
+import { Container } from '@dojo/widget-core/Container';
+import { TodoApp } from './../widgets/TodoApp';
+import uuid from '@dojo/core/uuid';
+import { Store } from 'le-store/store';
+import {
+	addTodoProcessWithPost,
+	editTodoProcess,
+	toggleTodoProcess,
+	deleteTodoProcess,
+	toggleAllTodoProcess,
+	clearCompletedProcess,
+	saveTodoProcess,
+	todoInputProcess,
+	clearFailedProcess
+} from './../processes/todoProcesses';
+
+function getProperties(store: Store, properties: any) {
+	const { get, createExecutor } = store;
+
+	return {
+		clearFailed: createExecutor(clearFailedProcess),
+		addTodo: createExecutor(addTodoProcessWithPost, (label: string) => {
+			return { id: uuid(), label, completed: false, loading: true };
+		}),
+		todoInput: createExecutor(todoInputProcess),
+		removeTodo: createExecutor(deleteTodoProcess),
+		toggleTodo: createExecutor(toggleTodoProcess),
+		toggleTodos: createExecutor(toggleAllTodoProcess),
+		clearCompleted: createExecutor(clearCompletedProcess),
+		editTodo: createExecutor(editTodoProcess),
+		saveTodo: createExecutor(saveTodoProcess),
+		currentTodo: get('/currentTodo'),
```

Comment:
The `get` function provides access to the stores state via a "path" (based on JSON Pointer). This is the only mechanism to access state.